client/src/views/common/BoostView.tsx

```diff
@@ -32,61 +42,86 @@ function onDeleteConfig(configType: ConfigT, name: string) {
   console.log(`name: ${name}`);
 }
 
-// Only dummy data
-const baseConfigs: BoostConfig[] = [
-  {
-    type: 'powerPlan',
-    options: [
-      { fileName: 'my_plan_1.json', displayName: 'my_plan_1' },
-      { fileName: 'this_one_gets_you_to_144.json', displayName: '144' },
-    ],
-    active: { fileName: 'my_plan_1.json', displayName: 'my_plan_1' },
-  },
-  {
-    type: 'rider',
-    options: [
-      { fileName: 'al.json', displayName: 'AL' },
-      { fileName: 'charles_rider.json', displayName: 'charles' },
-    ],
-    active: { fileName: 'charles_rider.json', displayName: 'charles' },
-  },
-  {
-    type: 'bike',
-    options: [
-      { fileName: 'blacksmith.json', displayName: 'Black Smith' },
-      { fileName: 'wombat.json', displayName: 'Wombat' },
-      { fileName: 'precilla.json', displayName: 'Precilla' },
-    ],
-    active: { fileName: 'wombat.json', displayName: 'Wombat' },
-  },
-  {
-    type: 'track',
-    options: [
-      { fileName: 'ford.json', displayName: 'Ford' },
-      { fileName: 'holden_track.json', displayName: 'Holden' },
-      { fileName: 'battle_mountain.json', displayName: 'Battle Mountain' },
-    ],
-    active: { fileName: 'ford.json', displayName: 'Ford' },
-  },
-];
-
 /**
  * Boost View component
  *
  * @returns {React.Component} Component
  */
 export default function BoostView() {
   // TODO: remove the hardcoded value for `distTravelled` with actual value read from MQTT
+  const [dist, setDist] = useState(0);
+  const [distOffset, setDistOffset] = useState<number | null>(null);
+  const [configs, setConfigs] = useState<BoostConfig[]>([
+    {
+      type: 'powerPlan',
+      options: [],
+    },
+    {
+      type: 'rider',
+      options: [],
+    },
+    {
+      type: 'bike',
+      options: [],
+    },
+    {
+      type: 'track',
+      options: [],
+    },
+  ]);
+
+  // fetch the reed distance from wireless module #3
+  const reedDistance = useSensorData(3, Sensor.ReedDistance, ReedDistanceRT);
+
+  useEffect(() => {
+    if (reedDistance) {
+      // Set distance to 2 decimal places
+      setDist(reedDistance);
+    }
+  }, [reedDistance]);
```

Comment:
So we see here that when `reedDistance` changes, `dist` is set to the value of `reedDistance`.
In other words, `dist` is the same as `reedDistance` but defaults to 0. This could be implemented more succinctly and understandable.